wiki/osu!supporter/fr.md

```diff
@@ -1,55 +1,152 @@
 ---
-outdated: true
-outdated_since: 6bc1537ef8d5ee9c08602c13108e15ce6f1ab524
+tags:
+  - support
+  - supporter
+  - supporter tag
+  - osu!supporter tag
```

Comment:
```suggestion
  - osu!supporter tag
  - tag supporter
  - tag osu!supporter
```

Reply:
je comprend pas cet ajout, tu inverses juste l'ordre des mots, ça ne change rien à la rechercher non ?

Reply:
a tag with multiple words is matched as a whole, so searching "tag supporter" would be guaranteed to lead directly to the article

Reply:
Je les mets juste en français pour le coup, car souvent les francophones qui ne connaissent pas les termes anglais vont les mettre en français

Reply:
@DoryanR ce que je voulais dire, c'est qu'inverser les tag ne va pas changer la recherche car c'est exactement les mêmes tag, ça me parait inutile de les rajouter étant donné qu'il n'y a pas de traduction à apporter

Reply:
Laissons comme tel alors, si ça te convient